whois-api/src/test/java/net/ripe/db/whois/api/rest/BatchUpdatesServiceTestIntegration.java

```diff
@@ -392,7 +392,8 @@ public void batch_update_delete_create_inetnum_success() {
                 .post(Entity.entity(whoisResources, MediaType.APPLICATION_JSON_TYPE), WhoisResources.class);
 
         assertThat(response.getWhoisObjects(), hasSize(2));
-        assertThat(response.getErrorMessages(), hasSize(2));
+        assertThat(response.getErrorMessages().stream().filter((message) -> message.getSeverity().equals("Info")).collect(Collectors.toList()), hasSize(2));
```

Comment:
Maybe *also* test that `getErrorMessages()` has a total size of 5, in addition to the Warning and Info checks